### PR TITLE
PP-53 Update ios-audiobooktoolkit to SwiftUI

### PR DESCRIPTION
**Pull request by vladimirfedorov**
**What's this do?**
Updates references and audiobook player initialisation of the ios-audiobooktoolkit module.

**Why are we doing this? (w/ Notion link if applicable)**
Updating `ios-audiobooktoolkit` to SwiftUI [PP-53](https://ebce-lyrasis.atlassian.net/browse/PP-53).

**How should this be tested? / Do these changes have associated tests?**
The new version of Palace audiobook player should work as expected.

**Dependencies for merging? Releasing to production?**
- [ios-audiobooktoolkit PR #50](https://github.com/ThePalaceProject/ios-audiobooktoolkit/pull/50)
- [ios-drm-audioengine PR #18](https://github.com/ThePalaceProject/ios-drm-audioengine/pull/18)

**Does this include changes that require a new Palace build for QA?**
Yes

**Has the application documentation been updated for these changes?**
Code documentation

**Did someone actually run this code to verify it works?**
@vladimirfedorov 

[PP-53]: https://ebce-lyrasis.atlassian.net/browse/PP-53?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Palace/MyBooks/TPPMyBooksDownloadCenter.m
Comment (mauricecarrier7) [inline Palace/MyBooks/TPPMyBooksDownloadCenter.m:1]: This class has been migrated to Swift now

Comment (vladimirfedorov) [reply Palace/MyBooks/TPPMyBooksDownloadCenter.m:1]: Nice catch, a relic from the pre-migration times )

Comment (vladimirfedorov) [reply Palace/MyBooks/TPPMyBooksDownloadCenter.m:1]: Also an interesting one, I don't have locally

Comment (vladimirfedorov) [reply Palace/MyBooks/TPPMyBooksDownloadCenter.m:1]: `git rm` solved the issue

**Review by mauricecarrier7 — APPROVED** [review]

**Pull request merged**